packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -125,4 +126,15 @@
             width: 657px;
         }
     }
+    &__data-list {
+        flex: 1;
+
+        &__data-list-body {
+            flex: 1;
+        }
+
+        .data-list__body-wrapper {
+            flex: 1;
+        }
+    }
```

Comment:
Would this work?

```suggestion
    &__data-list {
        flex: 1;

        &__data-list-body {
            flex: 1;
            
           &-wrapper {
               flex: 1;
        }
      }
    }
```

Reply:
No, in this case I want to target a `.data-list__body-wrapper` which is a (grand)child of `advertiser-page__data-list`.

<img width="385" alt="Screenshot 2020-11-20 at 14 52 35" src="https://user-images.githubusercontent.com/44427668/99768618-08c59580-2b40-11eb-8fd7-ba686a5bd7b9.png">